packages/features/ee/workflows/lib/constants.ts

```diff
@@ -64,6 +64,9 @@ export const DYNAMIC_TEXT_VARIABLES = [
   "reschedule_url",
   "rating_url",
   "no_show_url",
+  "attendee_timezone",
+  "event_time_in_attendee_timezone",
+  "event_end_time_in_attendee_timezone",
 ];
 
 export const FORMATTED_DYNAMIC_TEXT_VARIABLES = ["event_date_", "event_time_", "event_end_time_"];
```

Comment:
let's add "event_time_in_attendee_timezone" event_end_time_in_attendee_timezone" here too, then date formatting will work for these too